src/app/components/Filters/SelectedFilters.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  isEmpty as _isEmpty,
+  mapObject as _mapObject,
+  extend as _extend,
+  reject as _reject,
+} from 'underscore';
+
+import Actions from '../../actions/Actions';
+import appConfig from '../../../../appConfig';
+import { ajaxCall } from '../../utils/utils';
+
+const XCloseIcon = () => (
+  <svg
+    aria-hidden="true"
```

Comment:
Per button label discussion, remove `aria-hidden="true"` attribute and revise `title` attribute to "Remove Filter" instead of "Close Icon".

@seanredmond

Reply:
Will update. Make a note of that in the Design Toolkit as well if it's needed there, too.

Reply:
Updated.